tests/test_field.py

```diff
@@ -517,6 +517,37 @@ def test_create_and_update_field_definition(server_type):
     assert fieldDef.location == locations.nodal
 
 
+def test_field_definition_quantity_type(server_type):
+    fieldDef = FieldDefinition(server=server_type)
+
+    # Testing the setter
+    qt = "my_quantity_type"
+    fieldDef.set_quantity_type(qt)
```

Comment:
is it rather an add_entity_type?

Reply:
@cbellot000 

I used that name in order to be consistent with the C API that uses :

``` C++
ansys::dpf::FieldDefinition fd;
fd.setQuantityType()
```
But I agree that **add**_quantity_type would make more sense than set_quantity_type

... but - regarding one of your other comment where you suggest to have just one property for quantity_type - I would say that we should not define a setter using the decorator for quantity_type otherwise it would be confusing (because the setter would actually _add_ a quantity type, which is counter intuitive)

What I'm saying: 

We should have 

``` python
@property 
def quantity_type(self): 
   # Returns all quantity_types and makes quantity_type as a property

def add_quantity_type(self, quantity_type):
   # Add a new quantity_type
```

But we should NOT have:

``` python
@quantity_type.setter
def quantity_type(self,qt): 
   # adds a quantity type
```

Would that be fine with you ?